samples/peripheral/lpuart/sample.yaml

```diff
@@ -0,0 +1,9 @@
+sample:
+  name: Low Power UART
+  description: Low Power UART sample
+tests:
+  test_build:
+    build_only: true
+    build_on_all: true
+    platform_whitelist: nrf52840dk_nrf52840 nrf9160dk_nrf9160
```

Comment:
please verify if this list is correct ?

Should those boards be included ?
`nrf5340pdk_nrf5340_cpunet`, `nrf52dk_nrf52832`

Reply:
run and added `nrf52dk_nrf52832`, need to verify nrf53

Reply:
add nrf53 app as well.